adal/src/androidTest/java/com/microsoft/aad/adal/StorageHelperTests.java

```diff
@@ -249,6 +250,7 @@ public void testVersion() throws GeneralSecurityException, IOException {
         }
     }
 
+    @Suppress
```

Comment:
It's generally recommended to file a bug and include the bug number somewhere near the ignored (or suppressed test). See [this PR](https://identitydivision.visualstudio.com/IDDP/_git/TestAutomation/pullrequest/113?_a=files&path=%2FAndroidTests) for an example. 

This helps others who come along later to look at the bug and attempt to fix it - or gives them context as to why it should remain ignored.